packages/app/components/settings/add-email.tsx

```diff
@@ -27,25 +28,26 @@ export const AddEmailModal = () => {
 
   const submitEmail = useCallback(
     async (email: string) => {
+      rudder?.track("Button Clicked", {
+        name: "Login with email",
```

Comment:
should this be named `Login with email`?

Reply:
Nope, want to keep it consistent with the tracking for the login (the idea being to track someone using an email via Magic)